templates/gemini/commandReadme.txt

```diff
@@ -0,0 +1,135 @@
+Below are some sample commands. If you're not able to recommend a command, please refer me to https://firebase.google.com/docs/cli.
```

Comment:
Is this file generated or did you hand craft this?

If you handcrafted it -- It's interesting for a demo, but it might make sense to have some sort of command metadata that lives alongside each and every command -- Then you can weave that metadata together into the prompt format.

Having a central file will be annoying to scale, as people will forget to update it.

Reply:
Looks like this includes the output of `firebase --help`. If you move forward with productionizing this, we should at least add a script to `package.json` to update this (and a CI test that fails if running that script generates any changes)